src/vsphere_cpi/lib/cloud/vsphere/cluster_config.rb

```diff
@@ -8,8 +8,8 @@ def initialize(name, config_hash)
       @config = config_hash
     end
 
-    def resource_pool
-      @config['resource_pool']
+    %w[vm_group host_group vm_host_affinity_rule_name resource_pool].each do |name|
```

Comment:
This is terse but to be honest given how short this file is already is doing just:

```ruby
def vm_group; @config['vm_group']; end
def host_group; @config['host_group']; end
...
```

Is better in this case in my mind.